### PR TITLE
preview

### PR DESCRIPTION
**Pull request by sash-257defi**

### CODE REVIEW

**Review by AlexShpak — COMMENTED** [review]
Looks ok to me. It has some issues, but it's not Sash's issues. There are 2 questions:
file assets/flex.min.css contains only commented lines
comments inside some .svg files, like
&lt;!-- Generator: Adobe Illustrator ...

**Comment by sash-257defi** [comment]
assets/flex.min is a flex grid. Commented is a bootstrap  license. I thought I should leave it there.

File: _sass/colors.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+$contentInverseColor: #FFF;
+$secondaryContentColor: #C2C2C2;
+$borderColor: #DEDEDE;
+$fillColor: #F2F2F2;
+$positiveColor: #004385;
+$darkPositiveColor: #013365;
+$negativeColor: #C03;
+$noticeColor: #FEF1CF;
+$chartColor: #6B8Fb9;
+$chartAreaColor: rgba(128,179,236,0.3);
```
Comment (ashkanx) [inline _sass/colors.scss]: @sashcy is this necessary to move the colours here? why don'y you use the classes from this file? https://github.com/ChampionFX/binary-style/blob/gh-pages/_sass/common.scss
if there is any other element, I suggest you add those elements to the style guide.

Comment (sash-257defi) [reply _sass/colors.scss]: Noted. 
This should be removed, its not used anyways. Just forgot to remove.

**Pull request merged**